benchmark/Benchmark/Main.hs

```diff
@@ -0,0 +1,35 @@
+module Benchmark.Main
+       (main
+       ) where
+
+import Gauge.Main (bench, bgroup, defaultMain, nf)
+import Text.Toml (parseTomlDoc)
+
+import Benchmark.Tomland (codec)
+import Toml.Parser (parse)
+
+import qualified Data.Text.IO as TIO
+import qualified Benchmark.Htoml as Htoml
+import qualified Toml.Bi.Code as Tomland
+
+
+-- | Benchmark.
+main :: IO ()
+main = do
+    txt <- TIO.readFile "./benchmark/benchmark.toml"
+    Right htomlVal <- pure $ parseTomlDoc "" txt
+    Right tomlandVal <- pure $ parse txt
```

Comment:
I think that `parse` functions should be reexported from corresponding modules as well. So here you will have just:

```haskell
Right htomlVal <- pure $ Htoml.parse txt
Right tomlandVal <- pure $ Tomland.parse txt
```